docs/contributors/advanced_cli_usage.md

```diff
@@ -112,7 +112,7 @@ docker build -t ghcr.io/hyperledger/firefly .
 ```
 
 ## Edit your `manifest.json` file
```

Comment:
Editing the manifest file is actually no longer necessary, because the FireFly CLI will default to this same behavior when you pass in a manifest file. I think we can remove the whole `## Edit your manifest.json file` section